packages/bsky/src/hydration/feed.ts

```diff
@@ -71,7 +71,15 @@ export type ThreadRef = ItemRef & { threadRoot: string }
 
 // @NOTE the feed item types in the protos for author feeds and timelines
 // technically have additional fields, not supported by the mock dataplane.
-export type FeedItem = { post: ItemRef; repost?: ItemRef }
+export type FeedItem = {
+  post: ItemRef
+  repost?: ItemRef
+  /**
+   * If true, overrides the `reason` with `app.bsky.feed.defs#reasonPin`. Used
+   * only in author feeds.
+   */
+  authorPinned?: boolean
+}
```

Comment:
This is the part I'm least sure about - is it ok to add this extra field here?

Reply:
Yeah this type of a bit funny and could honestly probably find a cozier spot. This type is only used as a _parameter_ for both hydration & views. So you should be good to add this extra property here. The lil note is appreciated 👍